### ISSUE TITLE
es-stats support commonjs

### ISSUE THREAD

**Issue by jeasonstudio** [JennieJi/ast-lab#20]
CommonJS is not friendly to static analysis, do you have any good way to make`es-stats` support commonjs?

**Comment by JennieJi** [comment]
Possible but a lot harder... if I can do, I will also do it in another package instead, the analysis is very different

**Comment by jeasonstudio** [comment]
Yep. I have tried to do the static analysis for commonjs default export keys, but too  many cases to deal with.

**Comment by JennieJi** [comment]
May I ask what are you trying to resolving? And whether it is possible to convert to es instead?
